src/jdataview.js

```diff
@@ -201,10 +201,6 @@ jDataView.Uint64 = Uint64;
 Uint64.prototype = {
 	valueOf: function () {
```

Comment:
Please move `valueOf` outside as you did with `toString` for consistency (and because I was stupid when writing this :) ).